src/index.html

```diff
@@ -5,8 +5,384 @@
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#smarphones" class="nav__link">
+                Smartphones
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#computers"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops &amp; Computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#gadgets" class="nav__link">Gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#tablets" class="nav__link">Tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#photo" class="nav__link">Photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#video" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="page__main main">
+      <div class="main__wrapper">
+        <div class="main__card card" data-qa="card">
+          <a href="#">
+            <img
+              src="./images/imac.jpeg"
+              alt="Product image"
+              class="card__img"
+            >
+          </a>
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__subtitle">Product code: 195434</p>
+          <div class="card__reviews">
+            <div class="stars--4 stars">
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__reviews-text small-text">Reviews: 5</p>
+          </div>
+
+          <div class="card__payment">
+            <p class="card__payment-subtitle ">Price:</p>
+
+            <p class="card__payment-price">$2,199</p>
+          </div>
+
+          <a
+            href="#buy"
+            class="card__btn big-text"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
+
+        <div class="main__card card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="Product image"
+            class="card__img"
+          >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__subtitle">Product code: 195434</p>
+          <div class="reviews">
+            <div class="reviews__stars stars--4 stars">
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+            </div>
```

Comment:
you should use the same classes for the same code snippets, that`s why you have lots of fails in the test

Reply:
https://prnt.sc/NTsD77oWNQIT